examples/quickstart/jupyter-notebooks/Python_API_Tutorial.ipynb

```diff
@@ -0,0 +1,1262 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ce2efaaa",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Learn the Druid Python API\n",
+    "\n",
+    "This notebook provides a quick introduction to the Python wrapper around the [Druid REST API](api-tutorial.ipynb). This notebook assumes you are familiar with the basics of the REST API, and the [set of operations which Druid provides](https://druid.apache.org/docs/latest/operations/api-reference.html). Here we focus on using Python to access those APIs rather than explaining the APIs themselves. The APIs themselves are covered in other notebooks that use the Python API.\n",
+    "\n",
+    "The Druid Python API is primarily intended to help with these notebook tutorials. It can also be used in a regular Python program, as long as the IPython dependencies are available.\n",
+    "\n",
+    "The Druid Python API is a work in progress. We add API wrappers as needed for the notebook tutorials. If you find you need additional wrappers, please feel free to add them, and post a PR to Apache Druid with your additions.\n",
+    "\n",
+    "The API provides two levels of functions. Most are simple wrappers around Druid's REST APIs. Others add additional code to make the API easier to use. The SQL query interface is a prime example: extra code translates a simple SQL query into Druid's `SQLQuery` object and interprets the results into a form that can be displayed in a notebook.\n",
+    "\n",
+    "We start by importing the `druidapi` package from the same folder as this notebook. The `styles()` calls adds some CSS styles needed to display results."
```

Comment:
```suggestion
    "To start, import the `druidapi` package from the same folder as this notebook. The `styles()` calls adds some CSS styles needed to display results."
```
Prefer to avoid "we". Favor "you" for address.